src/app/app-modules/nurse-doctor/case-sheet/general-case-sheet/doctor-diagnosis-case-sheet/doctor-diagnosis-case-sheet.component.css

```
@@ -1,112 +1,428 @@
+    .container {
+        padding: 0;
+        max-width: 100%;
+        color: #000000;
+    }
+
+    h2,
+    h3,
+    h4,
+    h5 {
+        color: #000000;
+        page-break-after: avoid;
```

Stylelint (17.3.0)

[error] Expected "page-break-after" to be "break-after" (property-no-deprecated)

(property-no-deprecated)